bin/ci-release.sh

```diff
@@ -6,7 +6,7 @@ NEW_VERSION=${TRAVIS_TAG#"v"}
 
 # Build the extension
 yarn install
-yarn build
+yarn build:syntax
```

Comment:
When is `yarn build:extension` executed? Is that part of `yarn vscode:publish`?

Reply:
Yes it is. From the [vsce docs](https://code.visualstudio.com/api/working-with-extensions/continuous-integration#travis-ci-automated-publishing) it seems that `yarn vscode:publish` (aka `vsce publish`) does the packaging before publishing, as it is the only command needed to publish.

Reply:
Ok